_episodes/04-performance.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Teaching as a Performance Art"
```

Comment:
I would really feel it a pity if we loose the concept 'performance' completely. It's not fully Improv either, as we do start from a script. Nor is it fully interactive as the recipients have little control over the next steps - although 'the audience plays a critical role in shaping the act' is true. I think this basically means a -1 from me.